src/libraries/Common/src/Interop/Windows/SspiCli/SSPIWrapper.cs

```diff
@@ -453,13 +457,13 @@ public static string QueryStringContextAttributes(ISSPIInterface secModule, Safe
                     return null;
                 }
 
-                string result = Marshal.PtrToStringUni(sspiHandle.DangerousGetHandle());
+                string result = Marshal.PtrToStringUni(sspiHandle.DangerousGetHandle())!;
```

Comment:
The `QueryStringContextAttributes` method returns `string?`, why not just declare `string? result` and remove the suppression?